content/api/hub/plugin_spin_kinesis_trigger.md

```diff
@@ -0,0 +1,77 @@
+title = "Kinesis Trigger"
+template = "render_hub_content_body"
+date = "2024-10-04T00:22:56Z"
+content-type = "text/plain"
+tags = ["trigger", "rust", "plugins"]
+
+[extra]
+author = "ogghead"
+type = "hub_document"
+category = "Plugin"
+language = "Rust"
+created_at = "2024-08-31T00:22:56Z"
+last_updated = "2024-10-04T00:22:56Z"
+spin_version = ">=v2.6.0"
+summary = "An experimental Kinesis trigger for Spin apps"
+url = "https://github.com/ogghead/spin-trigger-kinesis"
+keywords = "trigger, kinesis"
+
+---
+
+An experimental plugin that enables AWS Kinesis trigger for Spin applications.
+
+## Installing Plugin
+
+You can install the plugin using the following command:
+
+```sh
+spin plugins install --url https://github.com/ogghead/spin-trigger-kinesis/releases/download/canary/trigger-kinesis.json
+```
+
+## Installing Template
+
+You can install the template using the following command:
+
+```sh
+spin templates install --git https://github.com/ogghead/spin-trigger-kinesis
```

Comment:
```suggestion
spin templates install --update --git https://github.com/ogghead/spin-trigger-kinesis
```
optional update flag to always have the latest. Not a blocker though

Reply:
Good call!